boards/microbit_v2/README.md

```diff
@@ -61,12 +62,19 @@ Drag and drop the [tock-bootloader.microbit_v2.vv1.1.1.bin](https://github.com/t
 
 The board will reset and the bootloader should be running on it. To check whether it's working, press and hold the Button A while pressing the reset button. The Microphone LED should light up.
 
+### Using openocd
+Use the `make flash-bootloader` command to flash [Tock Bootloader](https://github.com/tock/tock-bootloader) to the board.
+
+```bash
+$ make flash-bootloader
+```
+
 ### Using probe-rs
 
 Use the `make flash-bootloader` command to flash [Tock Bootloader](https://github.com/tock/tock-bootloader) to the board.
 
 ```bash
-$ make flash-bootloader
+$ make flash-bootloader USE_PROBE_RS
```

Comment:
I think elsewhere in the tock ecosystem, we use environment variables for this kind of thing, i.e.

```
USE_PROBE_RS=1 make flash-bootloader
```

which is a bit nicer as folks can also just set an environment variable once and not have to remember to change the command invocation every time. It also makes the Makefile impl a little simpler, i.e. just the `ifdef` no `eval` `filter` etc